atomic_reactor/plugins/pre_add_help.py

```diff
@@ -70,16 +70,23 @@ def run(self):
             return result
 
         dockerfile = df_parser(self.workflow.builder.df_path, workflow=self.workflow)
-        labels = dockerfile.labels
+        parsed_labels = dockerfile.labels.copy()
+        labels = Labels(dockerfile.labels)
+        try:
+            _, name = labels.get_name_and_value(Labels.LABEL_TYPE_NAME)
+        except KeyError:
+            name = ""
+        try:
+            maintainer = parsed_labels.get('maintainer')
+        except KeyError:
+            maintainer = ""
```

Comment:
If `'maintainer'` is not present we'll end up with `None`. I think instead of the try/except block we want:
```
maintainer = parsed_labels.get('maintainer', '')
```

Reply:
Fixed